<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>631</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="image_stats_tab">
     <attribute name="title">
      <string>Image-wise Stats</string>
     </attribute>
     <widget class="QPushButton" name="upload_button">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>291</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
     <widget class="QLabel" name="image_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>411</width>
        <height>581</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="image_stats_label">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>160</y>
        <width>251</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
     <widget class="QLabel" name="image_eye_rect_label">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>330</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="next_image_button">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>80</y>
        <width>291</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Next Image</string>
      </property>
     </widget>
     <widget class="QLabel" name="image_eye_color_label">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>320</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Eye Color:</string>
      </property>
     </widget>
     <widget class="QLabel" name="image_hair_color_label">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>370</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Hair Color:</string>
      </property>
     </widget>
     <widget class="QLabel" name="image_hair_rect_label">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>370</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>image_label</zorder>
     <zorder>upload_button</zorder>
     <zorder>image_stats_label</zorder>
     <zorder>image_eye_rect_label</zorder>
     <zorder>next_image_button</zorder>
     <zorder>image_eye_color_label</zorder>
     <zorder>image_hair_color_label</zorder>
     <zorder>image_hair_rect_label</zorder>
    </widget>
    <widget class="QWidget" name="dataset_stats_tab">
     <attribute name="title">
      <string>Whole Dataset Stats</string>
     </attribute>
     <widget class="MplWidget" name="MplWidget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>481</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="dataset_stats_label">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>160</y>
        <width>261</width>
        <height>331</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="dataset_eye_color_label">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>90</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Average Eye Color:</string>
      </property>
     </widget>
     <widget class="QLabel" name="dataset_hair_color_label">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>130</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Average Hair Color:</string>
      </property>
     </widget>
     <widget class="QLabel" name="dataset_hair_rect_label">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>130</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="dataset_eye_rect_label">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>100</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
